website/sidebars.json

```diff
@@ -28,6 +28,7 @@
                 "release-notes"
             ],
             "User": [
+                "user/setting_up_mac_apple_silicon",
```

Comment:
shouldn't it be only for the developer ? you didn't add this particular file

Reply:
Yes, this needs to be removed. 😄

Reply:
Removed ;)

Reply:
Does not seem to be removed :-D